pybloom_live/test_pybloom.py

```diff
@@ -68,6 +68,17 @@ def _run():
             bloom_one.union(bloom_two)
         self.assertRaises(ValueError, _run)
 
+    def test_union_scalable_bloom_filter(self):
+        bloom_one = ScalableBloomFilter(mode=ScalableBloomFilter.SMALL_SET_GROWTH)
+        bloom_two = ScalableBloomFilter(mode=ScalableBloomFilter.SMALL_SET_GROWTH)
+        chars = [chr(i) for i in range_fn(97, 123)]
+        for char in chars[int(len(chars) / 2):]:
+            bloom_one.add(char)
+        for char in chars[:int(len(chars) / 2)]:
+            bloom_two.add(char)
+        new_bloom = bloom_one.union(bloom_two)
+        for char in chars:
+            self.assertTrue(char in new_bloom)
```

Comment:
When we ask filter if it has seen an element and filter answers yes, the yes answer may be false positive. This could be caused by the collision of the hash value although it happens very rarely in our case. So asserting `char in new_bloom` could possible be false positive too theoretically, and we do not know when the hash values get clashed, so using `in` keyword here causing the test to be nondeterministic. 

I am aware that some existing tests do the same thing, it is due to some historical reasons, I will fix them. 

`bloom_one` will get assigned with `['n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']`, and `bloom_two` will get assigned with ` ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm']`. 2 filters here have no dupe elements in themselves, and elements in one filter are completely differ from elements in another. I wonder would it be better if we check `new_bloom.count` is the sum of the counts of `bloom_one` and `bloom_two`?